packages/docs/operations-manual/key-management/key-rotation.md

```diff
@@ -0,0 +1,40 @@
+## Key Rotation
+
+If a signing key is lost or compromised, the Locked Gold Account key can authorize a new signing key to replace the old one. This prevents losing a signing key from becoming a catastrophic event and it is strongly recommended to regularly rotate keys to limit the impact of an undiscovered compromise.
```

Comment:
any general heuristics on schedule for key rotation? what is the tradeoff in having to use colder account keys?

Reply:
Not really, it depends on the operational complexity, security of account keys, etc. Maybe on the order of monthly from a security standpoint, sometime when you would need to access your account key anyway for something else